README.md

```diff
@@ -112,13 +112,13 @@ Karafka has following configuration options:
 | Option                        | Required | Value type        | Description                                                                                                |
 |-------------------------------|----------|-------------------|------------------------------------------------------------------------------------------------------------|
 | name                          | true     | String            | Application name                                                                                           |
-| redis                         | true     | Hash              | Hash with Redis configuration options                                                                      |
+| redis                         | false    | Hash              | Hash with Redis configuration options. It is required if inline_mode is true.                              |
```

Comment:
inlne_mode is off I guess ;) if it is on, we don't have to have redis settings at all